src/mnode/rpc/masternode.cpp

```diff
@@ -117,6 +117,10 @@ UniValue masternodelist(const UniValue& params, bool fHelp)
         std::map<COutPoint, CMasternode> mapMasternodes = masterNodeCtrl.masternodeManager.GetFullMasternodeMap();
         for (auto& mnpair : mapMasternodes) {
             CMasternode mn = mnpair.second;
+            if( mn.IsNewStartRequired() && ! mn.IsPingedWithin(masterNodeCtrl.MasternodeWeekBySeconds) ) 
```

Comment:
This will only skip MNs with the NewStartRequired expired status.
But the task was saying about new parameter to masternode & masternodelist RPC API to force show all MNs even with this expired status.